cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/internal/BootstrapCoordinator.scala

```diff
@@ -284,7 +285,7 @@ private[akka] class BootstrapCoordinator(discovery: ServiceDiscovery,
 
   private[internal] def ensureProbing(contactPoint: ResolvedTarget): Option[ActorRef] = {
     val targetPort = contactPoint.port.getOrElse(settings.contactPoint.fallbackPort)
-    val rawBaseUri = Uri("http", Uri.Authority(Uri.Host(contactPoint.host), targetPort))
+    val rawBaseUri = Uri(selfUri.scheme, Uri.Authority(Uri.Host(contactPoint.host), targetPort))
```

Comment:
Good to not hard-code `"http"` here.

Taking the URI of the self-contactpoint is an improvement, though I guess it might not be universal if there is additional infrastructure building blocks in between. Eventually perhaps it would be good to make this a configuration option.

Until that time this is a nice improvement, though - and as you already mentioned, it should not be a breaking change, since situations where the self-contact-point scheme is 'https' would not have worked before either.